packages/rocketchat-dolphin/i18n/en.i18n.json

```diff
@@ -0,0 +1,6 @@
+{
+  "API_Dolphin_URL" : "Dolphin URL",
+  "Accounts_OAuth_Dolphin" : "Dolphin Login",
+  "Accounts_OAuth_Dolphin_id" : "Dolphin Key",
+  "Accounts_OAuth_Dolphin_secret" : "Dolphin Secret"
```

Comment:
We don't need the word `Dolphin` repeated all over. Delete this file and use generic keys for the settings: URL, Login, Key, Secret